### ISSUE TITLE
Add resize feature

### ISSUE THREAD

**Issue by mambucodev** [kenshaw/iv#2]
sometimes the `iv` command takes too much, or goes out of viewport. could you eventually resize the image to fit in the terminal viewport, or add a flag/configuration to do it myself maybe?

**Comment by kenshaw** [comment]
Just an update, I have kept this in mind, as I need this myself for a lot of stuff. It is coming, but I don't have a specific ETA. Probably the next time I'm dealing with a large batch of images/files that are larger than my viewport ...

**Comment by clayrisser** [comment]
Would be nice if it could default to not be drawn wider than the terminal window.

**Comment by kenshaw** [comment]
Yes, I know, I really don't have the time to build this. This is fairly easy to build if someone wants to give it a go, the main issue is that you need to use get the pixel width of the screen, and then calculate it. Otherwise could do something like just setting the forced max width. As it is, my life is infinitely better now with `iv` in it, and it works just fine for basically everything I need it for.

**Comment by clayrisser** [comment]
@kenshaw I have built this feature, if you would be so kind to accept the pull request. Thank you for your work on this project.

**Comment by kenshaw** [comment]
@mambucodev Sorry, the claude agent finally returned. Took it 2 years! Now implemented.
